invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/MobileCommunitiesItem.js

```diff
@@ -59,6 +60,10 @@ export const MobileCommunitiesItem = ({ result }) => {
             </a>
           )}
         </Item>
+        <Item.Extra>
+          {i18next.t("Created in: ")}{" "}
+          {DateTime.fromISO(result.created).toLocaleString(i18next.language)}
```

Comment:
what kind of metadata is this displaying ? could you include a screenshot?

Reply:
It is now tested. I thought it was bugged but what I thought as a bug was indeed a feature.